contrib/dev/Vagrantfile

```diff
@@ -2,10 +2,22 @@
 # vi: set ft=ruby :
 
 require 'vagrant-openstack-provider'
+require 'vagrant-reload'
+require 'vagrant-vbguest'
```

Comment:
@hunchback My first versions made use of vagrant-vbguest but I chose to use the Fedora packages instead. Anyway, this will go away soon as the VirtualBox guest additions will be integrated into the mainline kernel.

Reply:
I think this was needed to run (on VirtualBox) - so what is your recommendation for now? Should `require 'vagrant-vbguest'` be left in?

Reply:
Installing the guest additions is required - for now (need to test with Fedora 28 - https://fedoraproject.org/wiki/Changes/VirtualBox_Guest_Integration).

You can either use the vbguest plugin or install the RPM packages from rpmfusion. There is no reason to do both

Reply:
@lebauce - so currently do you have any objection that we go with `vagrant-vbguest` (and remove the guest additions RPM installation ?

Reply:
> You can either use the vbguest plugin or install the RPM packages from rpmfusion. There is no reason to do both

ok - so please propose which is the preferred approach

Reply:
My preferred approach would be to test building the Vagrant box with the newly released Fedora 28 instead of Fedora 27 as its base - as there should then be nothing to do.

Reply:
ok, checking ...

Reply:
can't find LXD on f28: https://copr.fedorainfracloud.org/coprs/ganto/lxd/ - please advise